### PR TITLE
Remove code to launch Pre-Fall 2021 Portal course without program SFID

### PR DESCRIPTION
**Pull request by elenabaurkot**
# Description
We added code in the PR to [Launch course for Portal PAFs without SF program](https://github.com/bebraven/platform/pull/1179) to let us launch the course 'Pre-Fall 2021 Braven Accelerator - PAFs'. This was to bring all PAFs who took the Accelerator in Portal over to Canvas so we could shut Portal down. We didn't need a Salesforce Program ID because these PAFs were already linked to the SF Program they took the course with originally. This was a one-time job and the code should be removed because courses should be launched with a SF program ID. 

# Motivation and Context
Remove unnecessary code that allows for atypical behavior. 

Task: https://app.asana.com/0/1201131148207877/1203415275471853/f

### CODE REVIEW

File: app/controllers/courses_controller.rb [hunk 1]
```diff
@@ -110,35 +105,33 @@ def launch_new
...
+      fellow_course_name,
+      lc_source_course_id,
+      lc_course_name = params.require([
+                                        :salesforce_program_id,
+                                        :notification_email,
+                                        :fellow_source_course_id,
+                                        :fellow_course_name,
+                                        :lc_source_course_id,
+                                        :lc_course_name
+                                      ])
```
Comment (tomlovett) [inline app/controllers/courses_controller.rb:122]: Nitpick: If you start a new line with the `[` then Rubocop should align these fields more to the left

File: app/controllers/courses_controller.rb [hunk 2]
```diff
@@ -110,35 +105,33 @@ def launch_new
...
       end
+    rescue ActionController::ParameterMissing, ActionController::BadRequest => e
+      redirect_to launch_courses_path, alert: "Error: #{e.message}" and return
+    end
 
-      # Start the program launch job
-      LaunchProgramJob.perform_later(salesforce_program_id, notification_email, fellow_source_course_id, fellow_course_name, lc_source_course_id, lc_course_name)
+    # Start the program launch job
+    LaunchProgramJob.perform_later(salesforce_program_id, notification_email, fellow_source_course_id,
+                                   fellow_course_name, lc_source_course_id, lc_course_name)
```
Comment (tomlovett) [inline app/controllers/courses_controller.rb:132]: Nitpick: Maybe put this arguments on a newline, too, for formatting's sake.

**Review by tomlovett — APPROVED** [review]
LGTM!

A few nitpicks about formatting, but no worries if they don't work or you don't want to add them.

File: app/controllers/courses_controller.rb (hunk 1 above)
Comment (elenabaurkot) [reply app/controllers/courses_controller.rb:122]: deleting old emails and realized I just saw your approval check, but missed these comments and merged @tomlovett - for this one do yo mean move the `[` to a new line like below?
```
lc_course_name = params.require(
   [
```

File: app/controllers/courses_controller.rb (hunk 2 above)
Comment (elenabaurkot) [reply app/controllers/courses_controller.rb:132]: Should each argument have it's own line here @tomlovett?

File: app/controllers/courses_controller.rb (hunk 1 above)
Comment (tomlovett) [reply app/controllers/courses_controller.rb:122]: Yup! That should fix the formatting.

And no worries, this isn't important; someone will catch it next time we're working on it.

File: app/controllers/courses_controller.rb (hunk 2 above)
Comment (tomlovett) [reply app/controllers/courses_controller.rb:132]: I think Rubocop would probably be happy with

```
LaunchProgramJob.perform_later(
    salesforce_program_id, notification_email, fellow_source_course_id, fellow_course_name, lc_source_course_id, lc_course_name
)
```

**Pull request merged**